go/cli/mcap/README.md

```diff
@@ -162,3 +160,21 @@ The `mcap list` command can be used with chunks or attachments:
 To ensure the resulting binary is statically linked, build with `make`:
 
     make build
+
+### API Evolution Policy
+
+This tool uses [semantic versioning](https://semver.org) to indicate the scope of an update to
+users. In addition, we try to follow these guidelines when updating the functionality of the tool.
+
+Here "option" refers to either a subcommand (such as `mcap convert`) or a command-line argument
+(such as `mcap --help`).
+
+- When a option needs to be removed, it is first marked as deprecated. Once deprecated,
+  it will remain functional for at least two releases or one month,
```

Comment:
Should we simplify this to just "one month"?

Reply:
Yeah, seems overcomplicated in retrospect.